examples/voting-poll/sample-ui/README.md

```diff
@@ -0,0 +1,50 @@
+# Sample UI Project - Voting App
+
+## Description
+
+This is a demo application built using Next.js 14 and npm. The application allows users to create polls and vote on them using their Stargazer DAG wallet. Users can only vote if they have balance in their wallet.
+
+
+## Features
+
++ Create new polls
++ Vote on existing polls
++ Authentication via Stargazer DAG wallet
++ Check wallet balance before voting
+  
+## Prerequisites
+
++ Node.js (v18 or higher)
++ npm (v9 or higher)
++ Stargazer DAG wallet
+
+## Usage
+
+### Development server
+
+1. Install dependencies:
+
+```sh
+npm install
+```
+
+2. start the development server:
+
+```sh
+npm run dev
+```
+
+Open http://localhost:5431 with your browser to see the result.
```

Comment:
Note, ensure that your metagraph is running using `hydra status` before opening this app. It will fail if the metagraph is offline. 

(would be nice to have an error that says "cannot reach metagraph, check if it's online with hydra status" instead of the cannot fetch error)